docker-compose.yml

```diff
@@ -1,25 +1,26 @@
 version: "3.8"
-
+volumes:
+  aws-keys-volume:
 services:
   localstack:
     container_name: localstack
-    image: localstack/localstack:1.3.1
+    image: localstack/localstack:latest
```

Comment:
I'm a little wary of using `latest` here as localstack has released breaking changes in the past. Could we pin to the same major version as we use in the Github tests (v3)?